CHANGELOG.md

```diff
@@ -34,3 +34,957 @@
 ### BREAKING CHANGES
 
 * All output properties are now readonly and cannot be reassigned.
+
+# 23.0.0 (2025-06-24)
```

Comment:
This is a 1:1 copy from `docs/changelog.md` with the sole change of converting all H2 titles to H1, so it's consistent with the new changelog entries.

Reply:
LGTM